openconcept/utilities/math/__init__.py

```diff
@@ -1,4 +1,7 @@
 from .add_subtract_comp import AddSubtractComp
```

Comment:
OpenMDAO also has an (AddSubtractComp)[https://github.com/OpenMDAO/OpenMDAO/blob/master/openmdao/components/add_subtract_comp.py].  This seems like it could be a bit confusing, or maybe these do similar things?

Reply:
Didn't know this, and funny enough it was Ben who added it to OM. I'll replace OC's version with OM's

Reply:
I actually take it back. OpenConcept's version can handle addition/subtraction of a combination of scalar and vector values, while OpenMDAO's can't. For now let's stick with OpenConcept's and in a future update we can switch to OpenMDAO's if it is updated